arm/Microsoft.ServiceBus/namespaces/migrationConfigurations/deploy.bicep

```diff
@@ -9,10 +9,10 @@ param name string = '$default'
 @description('Required. Name to access Standard Namespace after migration')
 param postMigrationName string
 
-@description('Required. Existing premium Namespace ARM Id name which has no entities, will be used for migration')
+@description('Required. Existing premium Namespace ARM ID name which has no entities, will be used for migration')
 param targetNamespace string
```

Comment:
what is an "ARM ID name" ? Is this Resource ID? I also don't understand the sentence.

Reply:
That's a good question. It's actually from the official documentation

Reply:
I guess it is a resource ID. Should we assume that?

Reply:
Lets assume that an ARM ID is indeed a resource ID :D